include/oneapi/dpl/internal/random_impl/normal_distribution.h

```diff
@@ -46,15 +46,45 @@ class normal_distribution
     // Distribution types
     using result_type = _RealType;
     using scalar_type = internal::element_type_t<result_type>;
-    using param_type = typename ::std::pair<scalar_type, scalar_type>;
+    class param_type
+    {
+      public:
+        typedef normal_distribution<result_type> distribution_type;
```

Comment:
While there is no technical difference I would recommend to change `typedef` to `using` alias style for readability and consistency.